types/pino/index.d.ts

```diff
@@ -733,6 +733,7 @@ declare namespace P {
          * Log at `'fatal'` level the given msg. If the first argument is an object, all its properties will be included in the JSON line.
          * If more args follows `msg`, these will be used to format `msg` using `util.format`.
          *
+         * @typeparam T: the interface of the object being serialized. Default is object.
```

Comment:
`@typeparam` does not appear to be a valid tag according to the [Typedoc documentation](https://typedoc.org/guides/doccomments/#typeparam-param-name), assuming that's the system that parses these comments in the first place. Replace all instances with either `@typeParam` or `@template` please.

Reply:
Fixed